release-lockfile.spec.md

```diff
@@ -148,6 +148,34 @@ information as to help differentiate the two deployed instances in a meaningful
 way.
 
 
+### Package Name
+
+A string matching the regular expression `[a-zA-Z][-_a-zA-Z0-9]{0,213}`
+
+
+#### Content Addressable URI
+
+Any URI which contains a cryptographic hash which can be used to verify the
+integrity of the content found at the URI.
+
+It is **recommended** that tools support IPFS and Swarm.
+
+
+#### Chain Definition 
+
+This definition originates from BIP122 URI
```

Comment:
TODO: add link to BIP122

Reply:
question: Is anyone using this part of the spec currently?

Reply:
Yes but not in *"production"* so-to-speak.  Why do you ask?

Reply:
Just curious if anyone is coming around to the problem of "how do different teams use this for their development chains"?